WordPress/Classes/ViewRelated/Post/PostEditorAnalyticsSession.swift

```diff
@@ -38,7 +38,8 @@ struct PostEditorAnalyticsSession {
     func preview(template: String) {
         let properties = commonProperties.merging([ Property.template: template], uniquingKeysWith: { $1 })
 
-        WPAnalytics.track(.editorSessionTemplatePreview, withProperties: properties)
+        // TODO: remove unused parts of Gutenberg editor preview mode
```

Comment:
I removed the Gutenberg preview mode along with the Android one with https://github.com/WordPress/gutenberg/pull/29896
I haven't tested the iOS side of this yet though

Reply:
Thanks for drafting those! I can open an iOS PR to test that as well. :+1: